components/nimbus/ios/Nimbus/Nimbus.swift

```diff
@@ -277,37 +277,21 @@ public extension NimbusDisabled {
         return nil
     }
 
-    func initialize() {
-        return
-    }
+    func initialize()
```

Comment:
TIL.

Reply:
Eh, swiftformat was suggesting this (I think) but xcodebuild doesn't seem to like it so I'm tinkering on this now...   Turns out the empty braces are needed, it was the empty return that swiftformat wasn't happy with.